multinode-demo/bootstrap-validator.sh

```diff
@@ -84,6 +84,7 @@ args+=(
   --identity "$identity"
   --vote-account "$vote_account"
   --rpc-faucet-address 127.0.0.1:9900
+  --enable-rpc-get-confirmed-block
```

Comment:
Is it related to frozen accounts?

Reply:
Nope, pollution from another thing I was doing in parallel.  Removed